CONTRIBUTORS.md

```diff
@@ -52,6 +52,7 @@ Contributors to previous releases
 * Joel Denny, [ETI]
 * Roxana Diaconescu, [CalTech]
 * James Dinan, [Cray Inc.] / [The Ohio State University]
+* Thomas Van Doren, [Cray Inc.]
```

Comment:
I don't think this movement is right.  Thomas' last name is "Van Doren", Van isn't his middle name